gui/screens/onboarding.py

```diff
@@ -183,8 +184,14 @@ def wait_until_appears(self, timeout_msec: int = configs.timeouts.UI_LOAD_TIMEOU
         return self
 
     @allure.step('Sign in')
-    def sign_in(self):
+    def sign_in(self, attempts: int = 2):
         self._sign_in_button.click()
+        try:
+            return SplashScreen().wait_until_appears()
+        except:
+            assert attempts > 0, f'Next button was not clicked'
+            self.sign_in(attempts - 1)
+
```

Comment:
one extra blank line, two lines before class, please fix it

Reply:
sorry about that, fixed